WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -173,7 +179,8 @@
           <Border Classes="warning info">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses coinjoining coins." />
+              <TextBlock Text="Transaction uses coinjoining coins." 
+                         ToolTip.Tip="This may cause issues. For example, your transaction might be replaced by the coinjoin one. Consider waiting for the current coinjoin to be finished before sending this transaction." />
```

Comment:
```suggestion
                         ToolTip.Tip="This may cause your transaction to be replaced by the coinjoin. Consider waiting for the current coinjoin to be finished before sending this transaction." />
```